qan-api2/models/metrics.go

```diff
@@ -1026,3 +1026,102 @@ func (m *Metrics) ExplainFingerprintByQueryID(ctx context.Context, serviceID, qu
 
 	return res, errors.New("query_id doesnt exists")
 }
+
+const selectedQueryMetadataTmpl = `
+SELECT 	any(service_name), any(database), any(schema), any(username), 
```

Comment:
why are you using any?

Reply:
It used here like an aggregate functions for the string values with GROUP BY to group the result-set by one or more columns that could be the same or empty.